tests/app/broadcast_message/test_rest.py

```diff
@@ -583,7 +584,10 @@ def test_update_broadcast_message_status_updates_details_but_does_not_queue_task
     assert len(bm.events) == 1
     alert_event = bm.events[0]
 
-    assert len(mock_task.mock_calls) == 0
+    if current_app.config['NOTIFY_ENVIRONMENT'] == 'preview':
```

Comment:
For this test, the `current_app.config['NOTIFY_ENVIRONMENT']` is `test` and you are not changing it to `preview`, meaning that we aren't actually running the assertion on line 588.

You would need to parametrize this test to give it two environments (or maybe all the environments) and then set the config at the top for the appropriate assertion to be used

Reply:
ahh, good comment, now I think about it, changing the environment could break tests. Test environment is required to connect to the right db etc. So I have scrapped the assertion that didn't do anything.